site-packages/integralstor/urbackup.py

```diff
@@ -90,42 +87,29 @@ def get_log_data(logids=None):
     else:
         return log_data, None
 
-
 def get_logs(date_on=None, date_from=None, date_till=None):
-    logs = None
+    logs = []
     qry = ''
+    client_id_list = []
     try:
         urb_db, err = config.get_urbackup_db_path()
         if err:
             raise Exception(err)
         if not urb_db:
             raise Exception("Path not set")
 
-        if date_on is not None:
-            qry = 'select * from logs where strftime("%%Y-%%m-%%d",created) == "%s"' % (
-                date_on)
-        elif date_from is not None and date_till is not None:
-            qry = 'select * from logs where strftime("%%Y-%%m-%%d",created) between "%s" and "%s"' % (
-                date_from, date_till)
-        elif date_from == None and date_till == None:
-            qry = 'select * from logs'
-        elif date_from and date_till == None:
-            qry = 'select * from logs where strftime("%%Y-%%m-%%d",created) >= "%s"' % (
-                date_from)
-        elif date_from == None and date_till:
-            qry = 'select * from logs where strftime("%%Y-%%m-%%d",created) <= "%s"' % (
-                date_till)
-        logs, err = db.get_multiple_rows(urb_db, qry)
-        if err:
-            raise Exception(err)
+        qry = 'select id from clients'
+        client_id = db.get_multiple_rows(urb_db, qry)
```

Comment:
Since this returns multiple rows, the variable also by convention should be plural so use client_ids